firstvoices/backend/resources/media.py

```diff
@@ -66,3 +66,87 @@ def before_import_row(self, row, **kwargs):
     class Meta:
         model = Audio
         clean_model_instances = True
+
+
+class ImageResource(ControlledSiteContentResource):
+    original = fields.Field(
+        column_name="img_original",
+        attribute="original",
+        widget=ForeignKeyWidget(model=ImageFile),
+    )
+
+    title = fields.Field(
+        column_name="img_title",
+        attribute="title",
+    )
+
+    description = fields.Field(column_name="img_description", attribute="description")
+
+    acknowledgement = fields.Field(
+        column_name="img_acknowledgement", attribute="acknowledgement"
+    )
+
+    exclude_from_kids = fields.Field(
+        column_name="img_include_in_kids_site",
```

Comment:
Follow up to Sarah's comment: whether it's "on_kids_site" or "in_kids_site" should match for other media types as well